apb.yml

```diff
@@ -1,10 +1,11 @@
 version: 1.0
 name: metrics-apb
 description: Installs a metrics service based on Prometheus and Grafana
-bindable: False
+bindable: True
```

Comment:
@maleck13 considering that metrics find scrape targets via service discovery, when would any service / mobile-client want to bind to metrics? What is the use-case for binding to metrics?

Reply:
It is to allow the app to send metrics in